Week3/homework/4-step3.js

```diff
@@ -1,2 +1,15 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+function Dog() {
+    this.name = "Geogre",
+    this.color = "White",
+    this.numLegs = 4;
+  };
+
+  function Dog() {
```

Comment:
Why have you redefined the `Dog()` constructor function here? This is at minimum a bad practice and normally considered an error.

Note also that this file is not correctly indented. If you have the correct setting in VSCode this cannot normally happen. See [VSCode Tips](https://github.com/HackYourFuture/fundamentals/tree/master/VSCodeTips#customise-vscode-settings).